packages/config/config/devices/0x0260/hs1sa-z.json

```diff
@@ -12,5 +12,11 @@
 	"firmwareVersion": {
 		"min": "0.0",
 		"max": "255.255"
+	},
+	"metadata": {
+		"overwiew": "The smart smoke sensor detects smoke, and sends an alarm signals to alert of danger. When the smart smoke sensor detects smoke, it will send an alarm notification to your controller through wireless Z-Wave communication protocol.",
```

Comment:
See https://zwave-js.github.io/node-zwave-js/#/config-files/file-format?id=metadata, we have only a limited set of fields at the moment. Overview is not one of them.